kie-server-parent/kie-server-tests/pom.xml

```diff
@@ -647,7 +647,7 @@
                 </container>
                 <configuration>
                   <properties>
-                    <cargo.jvmargs>-Xmx1024m</cargo.jvmargs>
+                    <cargo.jvmargs>-Xmx4096m</cargo.jvmargs>
```

Comment:
This is quite significant change, is it really required to have so much memory for executable model or it is a leftover to be reverted?

Reply:
When is this amount of memory required? Is it in case when kjar is built by Maven from sources or when kjar is processed by Kie server or both?

Reply:
@sutaakar The compilation of the kjar with the executable model needs a lot of ram. So I'd say it's the second, it's the case when the kjar gets built by Maven

Reply:
Ok, as the compilation by Maven is done by test then it is ok to keep the same JVM args for application servers as it used to be, right?

Reply:
@sutaakar yes that shouldn't change. But the line you highlighted isn't the one used by cargo maven to build the kjar?

Reply:
No, Cargo is used just for running application servers.
The kjar sources are picked and compiled as part of test execution in failsafe plugin, the built kjars are placed in kie-server-testing-remote-repo dir, when Kie server starts (with Cargo plugin) its settings.xml points to kie-server-testing-server-local-repo dir as local repository and has kie-server-testing-remote-repo dir (with compiled kjars) as a remote repository.

Reply:
The highlited line sets the JVM arguments for application server.